packages/@aws-cdk/aws-s3-deployment/test/bucket-deployment.test.ts

```diff
@@ -203,6 +203,21 @@ test('deploy from a local .zip file', () => {
 
```

Comment:
Please also add an integration test that uses EFS - just to make sure the file system is created and attached correctly, you don't have to use an actual 500MB sized asset.

Reply:
Added an integration test as suggested.